vivisect/renderers/__init__.py

```diff
@@ -238,8 +239,10 @@ def renderLocation(self, mcanv, loc, name, isfunc, cmnt, extra):
 
             mcanv.addText(linepre, vatag)
             offset,bytes = self.vw.getByteDef(lva)
-            b = bytes[offset].encode('hex')
-            mcanv.addNameText(b, typename="undefined")
+            b = bytes[offset]
+            mcanv.addNameText(b.encode('hex'), typename="undefined")
+            if b in string.printable:
```

Comment:
i totally get you.  i waffled on this one.  the reason i went with the double ;'s is because i always try to keep the viv output something you can copy and paste and assemble.  
but i suppose this doesn't apply here, since there's no "db".  ok.  i'll fix this :)